src/equations/svaerd_kalisch_1d.jl

```diff
@@ -1,8 +1,7 @@
 @doc raw"""
-    SvaerdKalischEquations1D(; gravity_constant, eta0 = 0.0,
-                               alpha = 0.0,
-                               beta = 0.2308939393939394,
-                               gamma = 0.04034343434343434)
+    SvaerdKalischEquations1D(bathymetry_type = bathymetry_variable;
```

Comment:
Maybe add a note that currently only `bathymetry_variable` is supported? Adding new capabilities will not be breaking.

Reply:
Done in ce8f281f6f8a08c0cb4d4c5c254ae1a8c9a85aca.